src/popup/pages/home.tsx

```diff
@@ -138,7 +143,7 @@ const Home = () => {
                     )}
 
 
-                    <h3 className="text font-medium text-base leading-10">Tools:</h3>
+                    {renderToolsConditions.includes(true) && <h3 className="text font-medium text-base leading-10">Tools:</h3>}
 
                     <div className="tools flex flex-col gap-2">
```

Comment:
We can set a ref to this div and check the length of its children before rendering the tools header. This way we won't have to maintain the `renderToolsConditions` array separately.